resources/smpp/ra/src/main/java/org/restcomm/slee/resource/smpp/SmppSessionsImpl.java

```diff
@@ -495,4 +492,113 @@ public void fireUnknownThrowable(Throwable throwable) {
 
 	}
 
+	public class SenderThread extends Thread {
+
+		private Boolean running = true;
+		private LinkedBlockingQueue<SmppSendingTask> queue = new LinkedBlockingQueue<>();
```

Comment:
moved creation to run.
In case there will be such parameter for ESME , i will change the logic , however currently there is no such and i guess it will require much bigger changes ( management for ESME , etc should be modified ). It may be set as improvements requirement for future releases.